WebContent/bank/queryxpath.jsp

```diff
@@ -0,0 +1,57 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+
+<%
+/**
+ This application is for demonstration use only. It contains known application security
+vulnerabilities that were created expressly for demonstrating the functionality of
+application security testing tools. These vulnerabilities may present risks to the
+technical environment in which the application is installed. You must delete and
+uninstall this demonstration application upon completion of the demonstration for
+which it is intended. 
+
+IBM DISCLAIMS ALL LIABILITY OF ANY KIND RESULTING FROM YOUR USE OF THE APPLICATION
+OR YOUR FAILURE TO DELETE THE APPLICATION FROM YOUR ENVIRONMENT UPON COMPLETION OF
+A DEMONSTRATION. IT IS YOUR RESPONSIBILITY TO DETERMINE IF THE PROGRAM IS APPROPRIATE
+OR SAFE FOR YOUR TECHNICAL ENVIRONMENT. NEVER INSTALL THE APPLICATION IN A PRODUCTION
+ENVIRONMENT. YOU ACKNOWLEDGE AND ACCEPT ALL RISKS ASSOCIATED WITH THE USE OF THE APPLICATION.
+
+IBM AltoroJ
+(c) Copyright IBM Corp. 2008, 2013 All Rights Reserved.
+*/
+%> 
+    
+<jsp:include page="/header.jspf"/>
+
+<div id="wrapper" style="width: 99%;">
+	<jsp:include page="membertoc.jspf"/>
+    <td valign="top" colspan="3" class="bb">
+		<%@page import="com.ibm.security.appscan.altoromutual.util.ServletUtil"%>
+		<div class="fl" style="width: 99%;">
+			<h1>Search News Articles</h1>
+			<form id="QueryXpath" method="get" action="<%=request.getRequestURL()%>">
+			  Search our news articles database
+			  <br /><br />
+				<input type="hidden" id=content" name="content" value="queryxpath.jsp"/>
+				<input type="text" id="query" name="query" width=450 value="<%=(request.getParameter("query")==null)?"Enter title (e.g. Watchfire)":request.getParameter("query")%>"/>
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; Cross-site Scripting (XSS)

Unsanitized input from `an HTTP parameter` flows into `print`, where it is used to render an HTML page returned to the user. This may result in a Cross-Site Scripting attack (XSS).

<h6>
Line 36 | <a href="https://cwe.mitre.org/data/definitions/79.html">CWE-79</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 842</a> | <a href="https://learn.snyk.io/lesson/xss/">Learn more about this vulnerability</h6>

<details>

<summary><b>Data flow: 4 steps</b></summary>

`Step 1 - 4` https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/WebContent/bank/queryxpath.jsp#L36

</details>